content/tutorials/_index.md

```diff
@@ -3,7 +3,41 @@ title: Tutorials
 weight: 1
 ---
 
-This part of the docs contains some practical guides for getting starting with both Python and our kit.
+This part of the docs contains some practical guides for getting starting with both Python and the simulator.
 
 * [Basic Movement](basic-movement) - explains how to use the kit to operate a motor
 * [Python: A Whirlwind Tour](python-whirlwind-tour) - a guide to using Python, aimed towards those who haven't used it before
+* [A basic guide to using the simulator](simulator) - a guide to using the we-bot simulator
+
+## Installation
+
+### Prerequisites
+
+You need to download and install Webots (the download is around 1.5GB). This is the platform we run our simulation in. Whilst we recommend version “R2021a” of Webots, and will be running competition simulations using it, “R2020b-rev1” should still work.
+
+## Python Version
+
+You will also need Python installed.
+
+| Platform | Supported Python Version|
+| -------- | ----------------------- |
+| Windows | 3.9 (64 bit) |
+| macOS | 3.9 |
+| Linux | 3.9 |
+
+In the competition, python 3.9 will be used.
+There are a small number of [extenal libraries](https://github.com/sourcebots/docs/blob/sp2021/content/api/_index.md#included-libraries) which will be available to your robot code during the competition. For local development you will need to install these yourself.
```

Comment:
I think this is the correct way to link? (@RealOrangeOne please shout if I'm wrong!)
```suggestion
There are a small number of [extenal libraries](/content/api#included-libraries) which will be available to your robot code during the competition. For local development you will need to install these yourself.
```